public/static/page6.tsx

```diff
@@ -25,36 +28,38 @@ const NoWrapButton = styled(Button)`
 `;
```

Comment:
Now `white-space: nowrap` can be set as style preset, wondering if we can address that here.
Edit: feel free to ignore it, maybe is better for another ticket in which we change some of the code

Reply:
I will skip this one for now.